### DISCUSSION TITLE
Get the tool to launch on startup, minimized to tray

### DISCUSSION THREAD

**Discussion by Bonswally** [4JX/L5P-Keyboard-RGB#111] [General]
I got the tool to run at startup by putting a shortcut to it in the startup folder and making the shortcut start minimized in properties.  Automatically loads the profile by creating default.json

Now I'm getting greedy and want it to start hidden in the system tray, any ideas?

**Comment by onix89** [comment]
I use legion toolkit actions.

![image](https://github.com/4JX/L5P-Keyboard-RGB/assets/52876623/d34dd95b-a770-4086-95ce-5302f32d1e60)
